src/sentry/api/serializers/models/deploy.py

```diff
@@ -10,10 +10,10 @@
 class DeploySerializer(Serializer):
     def get_attrs(self, item_list, user, *args, **kwargs):
         environments = {
-            e.id: e
-            for e in Environment.objects.filter(
+            id: name
+            for id, name in Environment.objects.filter(
                 id__in=[d.environment_id for d in item_list],
-            )
+            ).values_list('id', 'name')
```

Comment:
This line suggested the name is the intended value here, not the `Environment` object:

https://github.com/getsentry/sentry/blob/45a22ff7128d3c7e5b483ac2aaa50fa3b61c26fe/tests/sentry/api/endpoints/test_release_deploys.py#L97